### PR TITLE
Maintaining sort by key

### PR DESCRIPTION
**Pull request by jstavh**
Around sorting in front page and sub pages.

### CODE REVIEW

**Comment by robinbb** [comment]
Let's have max review this short PR.

File: web/components/proposal-table.tsx [hunk 1]
```diff
@@ -91,12 +91,7 @@ const columns: ColumnDef<ProposalListParserOutcome[number]>[] = [
       </div>
     ),
     enableHiding: false,
-  },
-  {
-    accessorKey: 'category',
-    filterFn: (row, value) => {
-      return value.includes(row.original.category);
-    },
```
Comment (52) [inline web/components/proposal-table.tsx:99]: This removes the ability to filter by proposal category.
Was this wanted? @robinbb

Comment (52) [inline web/components/proposal-table.tsx:99]: This removes the ability to filter by proposal category.
Was this wanted? @robinbb

**Review by 52 — CHANGES_REQUESTED** [review]

File: web/components/proposal-table.tsx (hunk 1 above)
Comment (robinbb) [reply web/components/proposal-table.tsx:99]: No, this seems to be a typo that removes an entire brace-delimited stanza by accident. Please correct, @jstavh

Comment (robinbb) [reply web/components/proposal-table.tsx:99]: @52 Note that filtering by category does not currently work for me on https://mina.vote. What about you?

Comment (52) [reply web/components/proposal-table.tsx:99]: @robinbb Weird - it does work for me. Which browser are you using?

Comment (jstavh) [reply web/components/proposal-table.tsx]: @52 made this change in the recent commit

**Review by robinbb — CHANGES_REQUESTED** [review]

**Comment by robinbb** [comment]
@jstavh Consider merging in a recent 'main'.

**Review by robinbb — APPROVED** [review]

**Pull request merged**